packages/cli/test/unit/commands/open.test.ts

```diff
@@ -0,0 +1,261 @@
+import { join } from 'path';
+import { defaultProject, useProject } from '../../mocks/project';
+import { useTeams } from '../../mocks/team';
+import { useUser } from '../../mocks/user';
+import open from '../../../src/commands/open';
+import { client } from '../../mocks/client';
+import { useDeployment } from '../../mocks/deployment';
+
+const fixture = (name: string) =>
+  join(__dirname, '../../fixtures/unit/commands/open', name);
+
+describe('open', () => {
+  const originalCwd = process.cwd();
+  let teamSlug: string = '';
+
+  it('should open the dashboard', async () => {
+    const cwd = fixture('project');
+    try {
+      process.chdir(cwd);
+
+      useUser();
+      const team = useTeams('team_dummy');
+      teamSlug = team[0].slug;
```

Comment:
It looks like other tests depend on this one running. I suggest avoiding test dependencies because it makes them brittle and impossible to run in isolation.

Reply:
This was a good catch, thank you.